applications/style.css

```diff
@@ -350,6 +352,7 @@ body {
   font-size: 16px;
   padding: 6px 16px;
   margin-left: auto;
+  margin-top: 5%;
```

Comment:
why 5%?

Reply:
the text was overlapping on the view details button so i've added a margin for that

Reply:
@vinit717

Reply:
> @vinit717

Can You Check this pls